Math-Interactions-Overview.md

```diff
@@ -0,0 +1,49 @@
+There are currently three interactions that can be used for math-related questions in lessons:
+- AlgebraicExpressionInput
+- NumericExpressionInput
+- MathEquationInput
+
+## Code structure
+### Frontend
+The interaction-specific files are in `extensions/interactions`. Each interaction has its own directory. The general structure of this directory would be as follows:
+
+![interactions](https://user-images.githubusercontent.com/35144226/228078766-a0fb8b8a-a03b-4df1-add5-59725d41cdeb.PNG)
+
+The `core/templates/services` contains the following services related to the math interactions:
```

Comment:
You introduce the terms guppy and nerdamer here without explaining what they are; perhaps some elaboration is in order? I know you have a glossary below, but people haven't read that far yet. It may be worth having an opening section explaining that the math interactions depend on several third-party components, and giving an introduction to each of those dependencies.

Reply:
Done.